rbc/message.go

```diff
@@ -0,0 +1,59 @@
+package rbc
+
+// Request message
+type RBCMessage struct {
+	Payload interface{}
+}
+
+type (
+	ValueRequest struct {
+		RootHash []byte
+		Branch   []byte
+		Block    [][]byte
+	}
+
+	EchoRequest struct {
+		ValueRequest
+	}
+
+	ReadyRequest struct {
+		RootHash []byte
+	}
+)
+
+type RecvRequests interface {
+	Add(connId ConnId) error
+	Get(connId ConnId) interface{}
+	GetAll() []interface{}
+	Len() uint
+}
+
+// Received request
+type (
+	RecvValueRequests struct {
+		recv map[string]*ValueRequest
+	}
+
+	RecvEchoRequests struct {
+		recv map[string]*EchoRequest
+	}
+
+	RecvReadyRequests struct {
+		recv map[string]*ReadyRequest
+	}
+)
+
+type (
+	// InnerMessage used in channel when received message
+	InnerMessage struct {
```

Comment:
Why defined `InnerMessage`? I think this is redundant and don' neccessary

Reply:
It is wrapper RBCMessage of ACS(it has sender ID) and will be distinguished into `VAL`, `ECHO`, `READY`